client/main.cpp

```diff
@@ -67,6 +67,7 @@ void real_main()
 			        .manual = true,
 			        .visible = true,
 			        .compatible = true,
+			        .tcp_only = app.get_server_tcp_only(),
```

Comment:
not sure if this would be better to have under `service`

Reply:
It would be better, as we have to pass both to establish a connection